connect/runtime/src/test/java/org/apache/kafka/connect/runtime/WorkerTest.java

```diff
@@ -2015,6 +2019,33 @@ public void testAlterOffsetsSourceConnectorError() throws Exception {
         verifyKafkaClusterId();
     }
 
+    @Test
+    public void testNormalizeSourceConnectorOffsets() throws Exception {
+        Map<Map<String, ?>, Map<String, ?>> offsets = Collections.singletonMap(
+                Collections.singletonMap("filename", "/path/to/filename"),
+                Collections.singletonMap("position", 20)
+        );
+
+        assertTrue(offsets.values().iterator().next().get("position") instanceof Integer);
+
+        JsonConverter jsonConverter = new JsonConverter();
+        jsonConverter.configure(Collections.singletonMap(SCHEMAS_ENABLE_CONFIG, false), false);
+        when(plugins.newInternalConverter(eq(true), anyString(), anyMap()))
+                .thenReturn(jsonConverter);
+        when(plugins.newInternalConverter(eq(false), anyString(), anyMap()))
+                .thenReturn(jsonConverter);
```

Comment:
Can we use the utility method here?
```suggestion
        mockInternalConverters();
```

Reply:
Ah yeah, this is left over from before I modified the `mockInternalConverters` method in order for the other tests to pass.